contribute/document-preview.md

```diff
@@ -18,9 +18,17 @@ Pulsar documentation is built using [Docusaurus](https://docusaurus.io/). To pre
 To verify docs are built correctly before submitting a contribution, you should set up your local environment to build and display the docs locally.
 
 * Node >= 16.14
-* Yarn >= 1.5
+* Corepack installed and enabled (`corepack enable`)
 * Although you can use Linux, macOS, or Windows to build locally the Pulsar documentation, macOS is the preferred build environment as it offers the most complete support for documentation building.
 
+Installing prerequisites with [homebrew](https://brew.sh/) on MacOS or Linux:
+
+```shell
+brew install node
+brew install corepack
```

Comment:
Not sure about the `brew install` step.

Corepack should be already available on newer Node without involving external package managers. Let me check.

Reply:
Node.js v20.x.x (LTS)

<img width="681" alt="Screenshot 2024-03-14 at 12 45 05 PM" src="https://github.com/apache/pulsar-site/assets/9302460/32cbe62b-f72e-4e93-9738-ee8361f41fa0">

Same for v21.x.x

Reply:
Thanks, PTAL